front/app/modules/commercial/custom_idea_statuses/admin/containers/index.tsx

```diff
@@ -68,7 +64,10 @@ const StyledIconTooltip = styled(IconTooltip)`
 const IdeaStatuses = () => {
   const { data: ideaStatuses, isLoading } = useIdeaStatuses();
   const { mutate: updateIdeaStatus } = useUpdateIdeaStatus();
-  const { mutate: deleteIdeaStatus } = useDeleteIdeaStatus();
+  const customIdeaStatusesAllowed = useFeatureFlag({
```

Comment:
This could lead to a situation where the the feature flag is allowed but disabled and then users will still have access to the feature. We should display the upsell message on `allowed: false` but still not display the feature on `enabled: false`. I guess we need 2 checks here 🤔

Reply:
Until now, this tab was showing for all clients because of faulty implementation of the flag. So this is equivalent to allowed/enabled for all. This hasn't led to complaints afaik. The nature of the feature also tells me that clients will (and should) not want this feature to be disabled. If you don't want to customise the statuses, then don't click the buttons to do so. :) 

I can't think of why you would want not to show that tab in the settings. This is not like proposals, where disabling the features makes it disappear from the platform from the navigation, homepage, etc.

So TL;DR: I think that's fine and what we want.